test/ref_ops_test.cpp

```diff
@@ -8765,26 +8765,102 @@ TEST_CASE(transpose_dyn_test)
     EXPECT(migraphx::verify::verify_range(results_vector, gold));
 }
 
-#if 0
 TEST_CASE(unique_test)
 {
-    migraphx::program p;
-    auto* mm = p.get_main_module();
-    migraphx::shape s{migraphx::shape::float_type, {1, 12}};
-    std::vector<float> data = {
-      1.0f, 1.3f, 0.0f, -1.2f, 0.0f, -100.f, 200.f, 0.0f, 0.1f, 1.3f, 0.0f, 1.200f};
-    auto input = mm->add_literal(migraphx::literal(s, data));
-    auto ret   = mm->add_instruction(migraphx::make_op("unique"), {{"sorted", {0}}}, input);
-    mm->add_return({ret});
-    p.compile(migraphx::make_target("ref"));
-    auto result = p.eval({}).back();
-    std::vector<int64_t> result_vector;
-    result.visit([&](auto output) { result_vector.assign(output.begin(), output.end()); });
-    std::vector<int64_t> gold = {0, 0, 0, 0, 1, 1, 1, 1, 0, 0, 0, 0, 0, 0, 1, 1, 0, 0,
-                                 1, 1, 0, 0, 0, 0, 0, 1, 0, 2, 0, 2, 0, 2, 0, 0, 0, 0};
-    EXPECT(migraphx::verify::verify_range(result_vector, gold));
+    auto create_program = [](int64_t axis, int64_t sorted, size_t test_sz) {
```

Comment:
Rebase with respect to latest develop branch. 
There has been some changes in ref ops test structure.